internal/builders/docker/commands.go

```diff
@@ -43,6 +49,8 @@ func DryRunCmd(check func(error)) *cobra.Command {
 		Use:   "dry-run [FLAGS]",
 		Short: "Generates and stores a JSON-formatted BuildDefinition based on the input arguments.",
 		Run: func(cmd *cobra.Command, args []string) {
+			defer printVerboseHint(inputOptions.Verbose)
```

Comment:
@asraa I added this hoping to get the hint printed out, but it does not work. Is it because `os.Exit(1)` is explicitly called in the `checkExit` function? Or am I doing something wrong?

Reply:
Yes. This works, but if I don't pass `--verbose`, I don't get "use --verbose to see detailed error". 

Or I can remove this and fix it later.

Reply:
I removed the hint for now :)